manipulator: 
 joint_state_controller: 
  type: joint_state_controller/JointStateController
  publish_rate : 50
 first_joint_position_controller : 
  type : effort_controllers/JointPositionController
  joint : first_joint
  pid : {p: 10, i: 0.1, d: 0} 
 second_joint_position_controller : 
  type: effort_controllers/JointPositionController
  joint: second_joint
  pid : {p: 10, i: 0.1, d: 0}
 third_joint_position_controller: 
  type: effort_controllers/JointPositionController
  joint: third_joint
  pid: {p: 10, i: 0.1, d: 0}

